plugins/catalog-backend-module-bitbucket-server/src/providers/BitbucketServerEntityProvider.ts

```diff
@@ -53,6 +62,11 @@ export class BitbucketServerEntityProvider implements EntityProvider {
   private readonly logger: Logger;
   private readonly scheduleFn: () => Promise<void>;
   private connection?: EntityProviderConnection;
+  private readonly catalogApi?: CatalogApi;
+  private readonly tokenManager?: TokenManager;
+  private readonly eventBroker?: EventBroker;
+  private eventConfigErrorThrown = false;
+  readonly TARGET_ANNOTATION: string;
```

Comment:
Do you need to expose this? And if you need to, why don't you use a getter for it?

Reply:
Accidental exposure.